src/client/pages/Path/Path.jsx

```diff
@@ -93,8 +93,8 @@ LessonCard.propTypes = {
   type: PropTypes.string.isRequired,
 };*/
 CourseCard.propTypes = {
-  pathId: PropTypes.objectOf(PropTypes.shape).isRequired,
-  courseId: PropTypes.objectOf(PropTypes.shape).isRequired,
+  pathId: PropTypes.string.isRequired,
+  courseId: PropTypes.string.isRequired,
   curriculum: PropTypes.objectOf(PropTypes.shape).isRequired,
```

Comment:
So if we'd want to go the "correct" route we'd need to supply a proper shape like React.PropTypes.shape({ potato: React.PropTypes.object, turkey: React.PropTypes.string })